test/functional/rpc_misc.py

```diff
@@ -11,6 +11,7 @@
     assert_equal,
     assert_greater_than,
     assert_greater_than_or_equal,
+    wait_until,
```

Comment:
unrelated style nit: If you want you can cherry-pick the change to the test framework from https://github.com/bitcoin/bitcoin/pull/19134/files#diff-64721c5ee64d44f7114d6d0d2226db4d and then call self.wait_until below instead of the import.